src/channel.ts

```diff
@@ -692,6 +698,10 @@ export class Channel<StreamChatGenerics extends ExtendableGenerics = DefaultGene
     } as Event<StreamChatGenerics>);
   }
 
+  _isTypingIndicatorsEnabled(): boolean {
+    return this.getClient().user?.privacy_settings?.typing_indicators?.enabled ?? true;
```

Comment:
I would move `!this.getConfig()?.typing_events` check here as well.

Reply:
makes sense